packages/zoe/src/contracts/callSpread/calculateShares.js

```diff
@@ -10,7 +10,7 @@ import { make100Percent, make0Percent } from './percent.js';
  * allocated to the long side of a call spread contract. price gives the value
  * of the underlying asset at closing that determines the payouts to the parties
  *
- * @type {CalculateShares} */
+  @type {CalculateShares} */
```

Comment:
```suggestion
  * @type {CalculateShares}
  */
```

Reply:
Not sure how this change got into this PR anyway. Probably a `yarn lint-fix` or `yarn format`?

In any case, fixed.